server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -153,8 +123,9 @@ public JacksonJsonProvider getJacksonJsonProvider(@Json ObjectMapper objectMappe
     return provider;
   }
 
-  private static Server makeJettyServer(@Self DruidNode node, ServerConfig config)
+  static Server makeJettyServer(DruidNode node, ServerConfig config)
   {
+    log.info("Creating Jetty server listening at port [%s] with config [%s]", node.getPort(), config);
```

Comment:
Yes I thought the same. Main reason of logging this message was to print information about what `ServerConfig` the server is using however the server starts in `initializeServer` which currently does not require `ServerConfig`. I guess I can just pass in the object for logging purposes.